README.md

```diff
@@ -126,6 +126,9 @@ We are always trying to improve our documentation, if you have suggestions or ru
 ## Deployment
 
 ##### Likely only useful for 18F team members
+
+Before deploying, install the [Cloud Foundry CLI](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) and the [autopilot plugin](https://github.com/concourse/autopilot#installation).
+
```

Comment:
I'd add another quick note that the `autopilot` plugin requires Go, and that if you don't have Go installed you can follow the steps here:  https://pages.18f.gov/dev-environment-standardization/languages/go/